iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -24,6 +24,8 @@
 
     protected boolean fictitious = false;
 
+    protected Set<String> aliases = new TreeSet<>();
```

Comment:
Is there a reason why you choose a TreeSet instead of a HashSet. Does the order important for you? Maybe for the XML round trip?

Reply:
I do not remember if it was for ensuring that roundtrip test work... Usually, I would sa that order is not important but it may be nice to keep aliases sorted, as you said for consistency in XML I/O. However, it may not impact a lot the performances as there is no reason, in current use case, that an Identifiable would have thousands of aliases...